modules/ROOT/pages/configuration/connectors.adoc

```diff
@@ -90,17 +89,17 @@ server.bolt.advertised_address=<server-name>:9000
 [[connectors-options-for-bolt-thread-pooling]]
 == Options for Bolt thread pooling
 
-See xref:performance/bolt-thread-pool-configuration.adoc[Bolt thread pool configuration] to learn more about Bolt thread pooling and how to configure it on the connector level.
+See xref:performance/bolt-thread-pool-configuration.adoc[Bolt thread pool configuration] to learn more about Bolt thread pooling and how to configure it on the network connector level.
 
 [[connectors-defaults-for-addresses]]
 == Defaults for addresses
 
-It is possible to specify defaults for the configuration options with `listen_address` and `advertised_address` suffixes, as described below.
-Setting a default value will apply to all the connectors, unless specifically configured for a certain connector.
+It is possible to specify defaults for the configuration options with `listen_address` and `advertised_address` suffixes.
+Setting a default value applies to all network connectors unless specifically configured for a certain connector.
```

Comment:
Should be something like:

If a default value is set, it will apply to all
or
Default value are applied to all

Reply:
I haven't edited the text. I just fixed the future tense here, as this is how it works.